engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/StartEventSubscriptionManager.java

```diff
@@ -96,15 +96,14 @@ private DeployedProcess findLastStartProcess(
     return null;
   }
 
-  private void openStartEventSubscriptions(
+  private void openStartEventSubscription(
```

Comment:
@lzgabel I believe you misunderstood the method I was talking about. The method you changed should stay the same. I was talking about the method on line 114.